sentry.properties

```diff
@@ -0,0 +1,5 @@
+defaults.url=https://sentry.io/
+defaults.org=fewlines
+defaults.project=connect-account
+auth.token=e010c0ab20964317bed616fed74e2a5e92157c919399452ea844d11989b143ab
```

Comment:
Should this one be displayed in the code like that?

Reply:
Good question. This was auto-generated from the wizard. What do you think @Fenntasy?

Reply:
Yeah well I couldn't find the info in their documentation. Presented differently, can we not put it like that?

Reply:
For now it doesn't matter because it is a private repo, but I fear this could be a problem later

Reply:
> For now it doesn't matter because it is a private repo

It's not.
I've made the changes, and added the variable in Heroku c8e85a9